code/script.js

```diff
@@ -1,28 +1,43 @@
 // Variables that point to selected DOM elements
-const chat = document.getElementById('chat');
+const chat = document.getElementById('chat')
+const form = document.getElementById('form')
+const inputWrapper = document.getElementById('input-wrapper')
+const input = document.getElementById('input')
```

Comment:
`const input = document.getElementById('input')` Hello Oscar! I know you said you hade some problems with these variables and that the inputWrapper solved it for you so I think you could delete this code when its not used (from what I can see).